059_digitOfThreeOccurs.js

```diff
@@ -0,0 +1,12 @@
+// 59. Number of times digit 3 occurs in each and every number from 0 to n
```

Comment:
The question is number of times digit 3 occurs which means
The numbers 3, 13, 23, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39 all contains the digit 3  if we take n = 40
the count of number of 3s is 14